### DISCUSSION TITLE
Windows exploit protection defaults

### DISCUSSION THREAD

**Discussion by Androidonator** [keepassxreboot/keepassxc#8192] [Ideas]
Hello,
do you think there is any way of implementing installer to set exploit protection settings for keepassxc process? I have set them on my own and they work, but since I have no experience with them. It was just trial and error until it worked. I have basically enabled everything except win32k calls and code integrity guard.
I think it would be a of a great value to community.
I think most of them are set on in windows by default, but I am certain settings are not.

**Comment by droidmonkey** [comment]
Can you be more explicit about what you are referring to needs to be enabled?
